CHANGELOG.md

```diff
@@ -3,6 +3,65 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.9.3
```

Comment:
The data grid release includes new features, so maybe we should bump a minor version instead?

Reply:
Thank you!